packages/blade/src/components/ListView/_decisions/decisions.md

```diff
@@ -0,0 +1,481 @@
+# List View Pattern API Decision
+
+> [!NOTE]
+>
+> Questions like what is pattern, why are we building these patterns, and scope of pattern are answered in [pattern-terminology.md](./pattern-terminology.md)
+
+List View Pattern is a UI pattern that includes Table, Filters, and Search of the table. It is a commonly used pattern on Razorpay dashboards to visualize data and help consumers filter / search through that data.
+
+![alt text](image-5.png)
+
+## API
+
+### With Layout Component
+
+```jsx
+<ListView>
+  <ListViewFilters
+    quickFilters={[
+      {
+        title: 'All',
+        onClick: () => {},
+      },
+      {
+        title: 'Pending',
+        onClick: () => {},
+      },
+    ]}
+    onSearchChange={() => {
```

Comment:
We need to make sure consumers are able to externally control all aspects of quickfilters/search etc. 

Quick filters, search, and dropdowns may be controlled or have initial values set via URL query parameters.

Reply:
- Search will be optional
- Counter, Badge, etc